### PR TITLE
Replace invis smite to clear

### PR DESCRIPTION
**Pull request by mayokek**
Tested and worked.
Resolves issue #959

### CODE REVIEW

**Comment by Adamx120** [comment]
I aggre with Purgeall, Potion clearall.

Invis smite is needed for invis potion spammers etc.

**Comment by Wild1145** [comment]
Looks good to me.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java [hunk 1]
```diff
@@ -11,19 +11,19 @@
 import org.bukkit.potion.PotionEffectType;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.BOTH)
-@CommandParameters(description = "Shows (optionally smites) invisisible players", usage = "/<command> (smite)")
+@CommandParameters(description = "Shows (optionally clears) invisisible players", usage = "/<command> (clear)")
```
Comment (rsajeey) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_invis.java]: Should be "invisible" instead of "invisisible".

**Comment by Wild1145** [comment]
Don't think we should be replacing the smite functionality as stated in the two associated issues.

**Comment by OnlineRepz** [comment]
@Wild1145, Prozza liked my idea to replace it. Smiting them for invis is just giving them a wrong idea.

**Comment by Wild1145** [comment]
@OnlineRepz Just because he liked the idea does not mean the idea is final / will be approved. I like the idea of knowing the national lottery numbers, but I have to guess them like everyone else.

**Comment by JeromSar** [comment]
Another PR which resolves the issue has been accepted, sorry!
Thanks for contributing.

https://github.com/TotalFreedom/TotalFreedomMod/pull/1011

**Pull request closed without merging**